index.js

```diff
@@ -302,16 +301,27 @@ class McEncryptionSchemePolyfill {
       return capabilities;
     }
 
-    // If we land here, the browser does _not_ support the encryptionScheme
-    // field.  So we install another patch to check the encryptionScheme field
-    // in future calls.
+    // If we land here, the browser does _not_ support the mediaKeySystemAccess
+    // field or the encryptionScheme field.  So we install another patch to
+    // check the mediaKeySystemAccess or encryptionScheme field in future calls.
     console.debug('McEncryptionSchemePolyfill: ' +
         'No native encryptionScheme support found. '+
         'Patching encryptionScheme support.');
     // eslint-disable-next-line require-atomic-updates
     navigator.mediaCapabilities.decodingInfo =
         McEncryptionSchemePolyfill.polyfillDecodingInfo_;
 
+    if (!mediaKeySystemAccess) {
+      const mediaKeySystemConfig =
+          McEncryptionSchemePolyfill.convertToMediaKeySystemConfig_(
+              requestedConfiguration);
+      capabilities.keySystemAccess =
+          await navigator.requestMediaKeySystemAccess(
+              requestedConfiguration.keySystemConfiguration.keySystem,
+              [mediaKeySystemConfig]);
+      return capabilities;
+    }
+
```

Comment:
Since this is taken care of in polyfillDecodingInfo_, you don't need to it here.  We chain to the polyfill immediately below this point.

Reply:
We'll call the native decodingInfo again in polyfillDecodingInfo_. Would it be better if we use the result of the first call directly?

Reply:
Not significantly, I would guess.  This will only happen once.  We will patch over it with the polyfill version, and the probe version will not be called again.

Reply:
The Player actually calls probeDecodingInfo() a few times before we settle on polyfillDecodingInfo_(), since the Player call decodingInfo in parallel and that's is async. I'll improve that in the future if we see the latency as an issue.

Reply:
Ah, you make a good point.